org.phoenicis.playonlinux.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
I assume it's x86_64 only – look [here](https://github.com/flathub/flathub/wiki/App-Maintenance#limiting-the-set-of-architectures-to-build-on) how to limit target architectures.

Reply:
Like this?